tests/openebs-pool-container-failure_test.go

```diff
@@ -63,12 +64,13 @@ var _ = BeforeSuite(func() {
 	}
 
 	//Getting the Application Status
-	app, _ := client.AppsV1().Deployments("litmus").Get("percona", metav1.GetOptions{})
+	chaosNamespace := "litmus"
+	app, _ := client.AppsV1().Deployments(chaosNamespace).Get("percona", metav1.GetOptions{})
 	count := 0
 	for app.Status.UnavailableReplicas != 0 {
 		if count < 50 {
 			fmt.Printf("Percona Application is Creating, Currently Unavaliable Count is: %v \n", app.Status.UnavailableReplicas)
-			app, _ = client.AppsV1().Deployments("litmus").Get("percona", metav1.GetOptions{})
+			app, _ = client.AppsV1().Deployments(chaosNamespace).Get("percona", metav1.GetOptions{})
```

Comment:
you are defining chaosNamespace at line 91 then how you accessing this at 73 line. please check the other files also?

Reply:
pls see line 67 ... I've defined it there also (for beforesuite fun)
and other at line 91 for all BDDs
It was required as the scope of the variable ends as we change the fun.
And we can define it globally but can not initialize it globally so anyways we have to initialize it per fun so I choose to use (`:=`)

Reply:
done for all such variables